tst/EnergyPlus/unit/BaseboardElectric.unit.cc

```diff
@@ -0,0 +1,101 @@
+// EnergyPlus, Copyright (c) 1996-2022, The Board of Trustees of the University of Illinois,
```

Comment:
Several new and fleshed out unit test files here.  Note that the purpose of this PR is to scaffold out the unit tests, which should make it easier for developers to add targeted checks while they are doing normal development for defects and features.